modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -22,43 +22,156 @@ To provision bare-metal hosts to the cluster by using the BMO, you must set the
 
 . Create the `Secret` CR and the `BareMetalHost` CR.
```

Comment:
I think this item should be removed, it's confusing without further context.

Reply:
Updated as suggested.